changelogs/unreleased/gh-7483-ignore-mpack-types.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/box
+
+- Fixed the inability to insert an integral number into a double-formatted field (gh-7483).
```

Comment:
We use `*`, not in `-` in changelogs. Also, a line should be wrapped by 80 characters:

```md
## bugfix/box

* Fixed the inability to insert an integral number into a double-formatted
  field (gh-7483).
```

(Unfortunately we don't have an automated checker for changelogs yet, see #6996.)

Reply:
Done.